twitchobserver/__init__.py

```diff
@@ -0,0 +1,3 @@
+from twitchobserver.twitchobserver import TwitchChatObserver, TwitchChatEvent
```

Comment:
Updated the import statement.
Tested with Ubuntu 16.04.2 LTS, pip (8.1.1, Python 2.7) and pip3 (8.1.1, Python 3.5).
Tested with Windows 10, pip (9.0.1, Python 3.5).